airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -55,9 +55,13 @@ def request_params(self, next_page_token: Mapping[str, Any] = None, **kwargs) ->
         return params
 
     def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
-        data = response.json().get(self.data_field, [])
+        data = response.json()
 
-        for record in data:
+        data_field = self.data_field
+        if "errors" in data:
```

Comment:
Can you explain why we are making this choice? this seems wrong, we are outputting errors as the connector's output rather than handle these errors? shouldn't we either fail the sync or skip the errors if they are known, safe-to-skip errors?

Reply:
When we get `Export daily quota exceeded` error API to return 200 status and we handle error only during parsing. https://developers.marketo.com/rest-api/error-codes/#1029